.github/CHANGELOG.md

```diff
@@ -21,6 +21,58 @@ This release contains contributions from (in alphabetical order):
 
 <h3>New features since last release</h3>
 
+* The transform for the Jacobian of the classical preprocessing within a QNode,
+  `qml.transforms.classical_jacobian`, now takes a keyword argument `argnum` to specify
+  the QNode argument indices with respect to which the Jacobian is computed.
+  [(#1645)](https://github.com/PennyLaneAI/pennylane/pull/1645)
+
+  The output behaviour with `argnum=None` differs slightly for different interfaces,
+  following the respective output shape of the native `jacobian` equivalent:
+
+  ==============  =====================  ====================  ========================
+  Interface       ``argnum=None``        ``type(argnum)=int``  ``argnum=Sequence[int]``
+  ==============  =====================  ====================  ========================
+  ``'autograd'``  ``tuple(arrays)`` [1]  ``array``              ``tuple(array)``
+  ``'jax'``       ``array``              ``array``              ``tuple(array)``
+  ``'tf'``        ``tuple(arrays)``      ``array``              ``tuple(array)``
+  ``'torch'``     ``tuple(arrays)``      ``array``              ``tuple(array)``
+  ==============  =====================  ====================  ========================
+
+  [1] If all QNode argument are scalars, the tuple is unpacked and the one-dimensional Jacobian
+  arrays are stacked into one ``array``. If there only is one QNode argument, the tuple is 
+  unpacked as well. Both is due to the behaviour of ``qml.jacobian``.
+
+  An example for the usage of ``argnum`` is
+
+  ```python
+  @qml.qnode(dev)
+  def circuit(x, y, z):
+      qml.RX(qml.math.sin(x), wires=0)
+      qml.CNOT(wires=[0, 1])
+      qml.RY(y ** 2, wires=1)
+      qml.RZ(1 / z, wires=1)
+      return qml.expval(qml.PauliZ(0))
+
+  jac_fn = qml.transforms.classical_jacobian(circuit, argnum=[1, 2])
+  ```
+
+  The Jacobian can then be computed at specified parameters.
+  Note that unlike the behaviour in `tensorflow`, *all* QNode arguments have to be provided
+  to the transformed QNode, even if only a subset of the Jacobians is computed.
```

Comment:
I am tempted to also remove this sentence from the changelog, also for simplicity :)